ci/generate-pipeline-steps.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e -u -o pipefail
+if [[ -n "${DEBUG-}" ]]; then
+    set -x
+fi
+
+# Download the unreal-engine.version file from the GDK repo so we can run the example project builds on the same versions the GDK was run against.
+# This is not the pinnacle of engineering, as we rely on GitHub's web interface to download the file, but it seems like GitHub disallows git archive
+# which would be our other option for downloading a single file.
```

Comment:
This comment is not relevant for this repo
Maybe instead I would do a comment saying this is mostly copied from EP's CI, so that we at least know when we refactor this.